src/scripts/updateOED.bash

```diff
@@ -5,11 +5,8 @@
 # * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # *
```

Comment:
This file also has to be edited during setup, and should not have an MPL header. I should have caught that when the change was first made.

Also, I think `updateMamacMetersCron` would be a more accurate name for this file. `updateOED` could easily be understood to update the OED instance itself, rather than fetching new readings.

Reply:
When we add more functionality, we can rename it, but for now it should be named for what it does, and all it does right now is update mamac meters.